include/circt/Dialect/FIRRTL/AnnotationDetails.h

```diff
@@ -18,6 +18,12 @@
 namespace circt {
 namespace firrtl {
 
+//===----------------------------------------------------------------------===//
+// Common strings related to annotations
+//===----------------------------------------------------------------------===//
+
+constexpr const char *rawAnnotations = "rawAnnotations";
```

Comment:
nit constexpr char[] = ...;

Reply:
I won't change that here given that this is consistent with the existing `char *` usage in the rest of this file. Fine to swing this after the fact.